net.sourceforge.qtpfsgui.LuminanceHDR.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: net.sourceforge.qtpfsgui.LuminanceHDR
+runtime: org.kde.Platform
+runtime-version: "5.11"
+sdk: org.kde.Sdk
+command: luminance-hdr
+rename-desktop-file: luminance-hdr.desktop
+rename-appdata-file: luminance-hdr.appdata.xml
+rename-icon: luminance-hdr
+finish-args:
+  - "--socket=fallback-x11"
```

Comment:
fallback-x11 only makes sense if you also have wayland.

Reply:
Regarding this, as far as I've been told,

> Qt5 has a wayland backend so it can run on wayland

However neither @fcomida nor me running wayland. Thus we are not sure whether in modern distros

> qt5 apps are run through wayland, and how flatpak interacts with wayland.

@TingPing can you help us to figure that out?